infrastructure/src/main/java/org/corfudb/infrastructure/ManagementServer.java

```diff
@@ -113,15 +114,16 @@ public ExecutorService getExecutor() {
      */
     public ManagementServer(ServerContext serverContext) {
 
-        this.opts = serverContext.getServerConfig();
-        this.localEndpoint = this.opts.get("--address") + ":" + this.opts.get("<port>");
+        managementServerConfig = ManagementServerConfig.parse(serverContext.getServerConfig());
```

Comment:
Please keep the style consistent. It is generally a good practice to prefix member variables in a constructor with "this".

Reply:
I'm using "this" just when the variable in local scope shadows the fields. It seems using this adds some clutter without much value and so limiting it usage to when scope mandates its usage. Let me know if you feel strongly and will add them back.

Reply:
done